packages/venia-ui/lib/components/ProductListing/product.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { useProduct } from '@magento/peregrine/lib/talons/ProductListing/useProduct';
+import { Price } from '@magento/peregrine';
+
+import { mergeClasses } from '../../classify';
+import Kebab from '../MiniCart/kebab';
+import ProductOptions from '../MiniCart/productOptions';
+import Section from '../MiniCart/section';
+import Image from '../Image';
+import defaultClasses from './product.css';
+import { REMOVE_ITEM_MUTATION } from './productListing';
+
+const IMAGE_SIZE = 100;
+
+const Product = props => {
+    const { item } = props;
+    const talonProps = useProduct({
+        item,
+        removeItemMutation: REMOVE_ITEM_MUTATION
+    });
+    const {
+        handleEditItem,
+        handleRemoveFromCart,
+        handleToggleFavorites,
+        isRemoving,
+        product
+    } = talonProps;
+    const { currency, image, name, options, quantity, unitPrice } = product;
+
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const rowMask = isRemoving ? classes.mask : '';
+
+    return (
+        <li className={`${classes.root} ${rowMask}`}>
+            <Image
+                alt={name}
+                classes={{ image: classes.image, root: classes.imageContainer }}
+                width={IMAGE_SIZE}
+                resource={image}
+            />
+            <span className={classes.name}>{name}</span>
+            <ProductOptions
+                options={options}
+                classes={{ options: classes.options, optionLabel: {} }}
```

Comment:
An empty object is stringified to `[object Object]` in the html. You should just create an empty `.optionLabel` class in the css and pass it here.

Reply:
I know this API isn't pretty, but props for using it. At some point we'll make a better solution for this use case.

Reply:
I made this change in 0ac27f8, but vscode linting is complaining about the empty rule now. Is that something we enforce?

Reply:
No, VSC's linter is not something we care about, so an empty rule is fine. That said, you may safely delete empty rules. The component should not throw an error if you try to apply `classes.foo` but there is no `.foo {}` defined in the CSS module.